.github/workflows/build.yml

```diff
@@ -53,30 +52,87 @@ jobs:
       - name: Upload test coverage
         uses: codecov/codecov-action@v1
 
-      - name: Is Release?
-        if: startswith(github.ref, 'refs/tags/v')
-        run: echo "DEPLOY_PACKAGE=true" >> $GITHUB_ENV
+  publish:
+    needs: build
+    if: startswith(github.ref, 'refs/tags/v')
+    runs-on: ubuntu-latest
+    environment: production
+    env:
+      NODE_VER: 16.14.0
```

Comment:
So, we can build with multiple versions, but will publish with only one version?

Reply:
Build is for testing (and running UTs), but publish can work with any node version. We don't have `engines` specified in our package.json as of now https://docs.npmjs.com/cli/v9/configuring-npm/package-json?v=true#engines, so as long as the dependencies work, the published package should be able to run with any node. Added a comment to #494 to explore adding this to our package.json